### ISSUE TITLE
property "all-labels" is not defined in object type {} on actions/labeler@v4

### ISSUE THREAD

**Issue by EigoOda** [rhysd/actionlint#346]
```yaml
...
  labeler:
    runs-on: ubuntu-latest
    steps:
      - id: labeler
        uses: actions/labeler@v4
        with:
          repo-token: ${{ secrets.GITHUB_TOKEN }}
          configuration-path: .github/labeler.yaml
          sync-labels: ''
    outputs:
      labels: ${{ steps.labeler.outputs.all-labels }}
```

actionlint says 'property "all-labels" is not defined in object type {} [expression]'.

But the Following jobs (need labeler job outputs) are running correctly.
How can I resolve this warning?
Thanks.
